template/content/endpoints.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+module.exports = Endpoints;
+
+function Endpoints(app) {
+    if (!(this instanceof Endpoints)) {
+        return new Endpoints(app);
+    }
+
+    var self = this;
+
+    var channel = app.clients.appChannel;
+    var thrift = app.clients.tchannelThrift;
+
+    thrift.register(
+        channel, 'MyService::health_v1', app, self.health
+    );
+    thrift.register(
+        channel, 'MyService::get_v1', app, self.get
+    );
+    thrift.register(
+        channel, 'MyService::put_v1', app, self.put
+    );
+}
+
+Endpoints.prototype.health =
+function health(app, req, head, body, cb) {
```

Comment:
If a method is being called with a `this` that isn't bound to the `Endpoints` instance it shouldn't be on the prototype. It doesn't make sense for it to be on the prototype. Obviously it works but it's confusing and unnecessary. Just bind `this` correctly like everyone else